src/components/Button/Button.scss

```diff
@@ -0,0 +1,36 @@
+.button {
+  padding: 20px;

```

Comment:
I think we can remove `padding-top` and `padding-bottom` to be able to use `height: 40px` as in the `orange` variant; @MarcelloAragoni what do you think?